api_volontaria/apps/volunteer/tests/test_view_events.py

```diff
@@ -87,14 +88,9 @@ def test_create_new_event_as_admin(self):
             "end_time": LOCAL_TIMEZONE.localize(datetime(2100, 1, 15, 10)),
             "nb_volunteers_needed": 10,
             "nb_volunteers_standby_needed": 0,
-            "cell": reverse(
-                'cell-detail',
-                args=[self.cell.id],
-            ),
-            "task_type": reverse(
-                'tasktype-detail',
-                args=[self.tasktype.id],
-            )
+            "cell": self.cell.id,
+            "task_type": self.tasktype.id,
```

Comment:
We should not lose the URL just to add tags since it will break the homogeneity of the API JSON